types/canvasdimensions.ts

```diff
@@ -4,6 +4,6 @@ interface Dimensions {
 }
 
 export const getDimensions = (parentWidth: number): Dimensions => {
-    const baseIncrement = Math.max(Math.floor(parentWidth / 32), 1);
+    const baseIncrement = Math.max(parentWidth / 32, 1);
```

Comment:
The purpose of that floor is to make sure the canvas is resized to sensible dimensions:

<img width="1042" height="684" alt="Image" src="https://github.com/user-attachments/assets/31152931-9720-4f12-b762-36cc42ce8c60" />

If you want to override it just for embed pages you'll need to wire through an option for that.